docs/data-sources/aws_transit_gateway_attachment.md

```diff
@@ -20,6 +20,8 @@ data "hcp_aws_transit_gateway_attachment" "test" {
 }
 ```
 
+~> **Security Notice** This resource contains sensitive input. Please see this [list of recommendations](https://www.terraform.io/docs/language/state/sensitive-data.html) for storing sensitive information in Terraform.
+
```

Comment:
I don't believe the data source actually contains any sensitive inputs, so this message might be confusing

Reply:
Oops you're right, thought the ARN might be an output